gatling-core/src/main/scala/io/gatling/core/json/Json.scala

```diff
@@ -45,7 +49,7 @@ object Json {
   }
 
   private def writeString(s: String, rootLevel: Boolean) = {
-    val escapedLineFeeds = s.replace("\n", "\\n")
+    val escapedLineFeeds = objectMapper.writeValueAsString(s).replaceAll("^\"|\"$", "")
```

Comment:
I agree with all of above. The only way to get out of this mess is to copy the escaping mechanism from Jackson to gatling. Or to find one in Scala.